packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -536,9 +536,9 @@ export class ProposedTransformationExplorer {
             diffModel.saveChanges()
             telemetry.codeTransform_submitSelection.emit({
```

Comment:
Should be in a `run` block, have been slowly working on converting all of our metrics to use `run` blocks, mostly done but the old `emit` calls are still lingering around in a few places like here